portality/forms/application_forms.py

```diff
@@ -1318,7 +1318,14 @@ class FieldDefinitions:
         ],
         "attr": {
             "class": "input-xlarge unstyled-list"
-        }
+        },
+        "contexts": {
+            "admin": {
+                "widgets": [
+                    "autocheck",  # ~~^-> Autocheck:FormWidget~~
```

Comment:
The `context` fields overwrite the base fields, so in this case this would remove the `trim_whitespace` and `multiple_field` widgets, and replace them with just the `autocheck` field.  Instead, this should list all 3 required widgets.